client/header/activity-panel/index.js

```diff
@@ -277,6 +280,47 @@ export class ActivityPanel extends Component {
 		}
 	}
 
+	closedHelpPanelHighlight() {
+		const { userPreferencesData } = this.props;
+		recordEvent( 'help_tooltip_click' );
+		if (
+			userPreferencesData &&
+			userPreferencesData.updateUserPreferences
+		) {
+			userPreferencesData.updateUserPreferences( {
+				help_panel_highlight_shown: 'yes',
+			} );
+		}
+	}
+
+	shouldShowHelpTooltip() {
+		const {
+			userPreferencesData,
+			trackedCompletedTasks,
+			query,
+		} = this.props;
+		const { task } = query;
+		const startedTasks =
+			userPreferencesData &&
+			userPreferencesData.task_list_tracked_started_tasks;
+		const highlightShown =
+			userPreferencesData &&
+			userPreferencesData.help_panel_highlight_shown;
+		if (
+			task &&
+			highlightShown !== 'yes' &&
+			( startedTasks || {} )[ task ] > 1 &&
+			! trackedCompletedTasks.includes( task )
+		) {
+			if ( ! this.recordedHelpTooltip ) {
+				this.recordedHelpTooltip = true;
+				recordEvent( 'help_tooltip_view' );
```

Comment:
This should probably be in the tooltip component, no?

Reply:
Not necessarily, given the tooltip is more abstract, but it isn't the nicest place. I added a `onShow` callback to the tooltip, and calling it from there.